generators/server/templates/src/test/java/package/web/rest/_AccountResourceIntTest.java

```diff
@@ -146,16 +147,20 @@ public void testGetUnknownAccount() throws Exception {
     @Test<% if (databaseType == 'sql') { %>
     @Transactional<% } %>
     public void testRegisterValid() throws Exception {
-        UserDTO u = new UserDTO(
+        ManagedUserDTO u = new ManagedUserDTO(
```

Comment:
Very small improvement : even if it was already here before, could you take the opportunity of your PR to rename the variable u to user in this file? I found 1 letter variable misleading and hurtful to the code readability.